rotkehlchen/tests/api/test_users.py

```diff
@@ -549,16 +582,16 @@ def test_user_password_change(rotkehlchen_api_server, username, db_password):
         check_proper_unlock_result(result)
 
     assert rotki.user_is_logged_in is True
-    users_data = check_user_status(rotkehlchen_api_server)
-    assert len(users_data) == 1
-    assert users_data[username] == 'loggedin'
+    users_data_2 = check_user_status(rotkehlchen_api_server)  # type: ignore
```

Comment:
why rename it here? I don't see this used elsewhere. Also what is the type ignore for? You need to be stating these things when adding a type ignore.